### DISCUSSION TITLE
How to export calendar into ics files

### DISCUSSION THREAD

**Discussion by JoeyZhu** [nextcloud/all-in-one#2743] [Questions] [answered]
using nextcloud snap install or native install, there is a tool (https://codeberg.org/BernieO/calcardbackup.git) can easily export calendar data and backup. How to do with AIO version? Thanks!

**Comment by szaimen** [comment, marked as answer]
Hi, best would be to probably use this docker image: https://github.com/waja/docker-calcardbackup

In case of AIO the nextcloud config is located inside the nextcloud_aio_nextcloud volume and the database is in the nextcloud-aio-database container.
